frontend/src/components/Comment/PostComment.js

```diff
@@ -27,15 +35,13 @@ class PostComment extends React.Component {
         });
     }
 
-    render() {
-        return (
-            <form onSubmit={this.comment.bind(this)}>
+    return (
+        <div>
+            <form onSubmit={comment}>
                 <div className="comment-form-div">
-                    <input name="comment" className="comment-input"/><button>Comment</button>
+                    <textarea value ={textAreaText} name="comment" className="comment-input" autoComplete="off" onChange={handleTyping}/><button className="btn">Post</button>
```

Comment:
picky but, add a space after the `=` (or remove the space before it) :stuck_out_tongue: